sycl/test/basic_tests/event_profiling_info.cpp

```diff
@@ -1,5 +1,8 @@
 // RUN: %clang -std=c++11 -fsycl %s -o %t.out -lstdc++ -lOpenCL -lsycl
-// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+//
+// Profiling info is not supported on host device so far.
+// RUNx: env SYCL_DEVICE_TYPE=HOST %t.out
```

Comment:
I suggest removing RUN command.

Reply:
We use RUNx: approach in many tests. Should we change it everywhere?

Reply:
Makes sense to me.
These lines are has no value and can be confusing.
It's quite easy to miss the subtle difference between the commands which are executed and modified version which is ignored.